app/code/Magento/InventoryCatalogSearch/Plugin/Search/FilterMapper/AdaptStockStatusFilterPlugin.php

```diff
@@ -55,21 +57,28 @@ class AdaptStockStatusFilterPlugin
      */
     private $defaultStockProvider;
 
+    /**
+     * @var StockConfigurationInterface
+     */
+    private $configuration;
```

Comment:
I believe this has to be named as $stockConfiguration, once you use this name in constructor.